Dockerfile

```
@@ -7,14 +7,12 @@ WORKDIR /app
 
 COPY package.json .
 
-RUN apk add --no-cache --virtual .build-dependencies git && \
-    sed -i -e 's/"mongojs": "0.18.2"/"mongojs": "2.4.0"/g' package.json && \
-    yarn install && \
-    yarn cache clean && \
+RUN apk add --no-cache --virtual .build-dependencies git python make && \
```

python & make packages are not used (we can build and run the container without them) 
```suggestion
RUN apk add --no-cache --virtual .build-dependencies git && \
```